app/jobs/fs/sync_all_boxes_job.rb

```diff
@@ -0,0 +1,12 @@
+module Fs
+  class SyncAllBoxesJob < ApplicationJob
+    def perform
+      Fs::Box.where(syncable: true).find_each do |box|
+        SyncBoxJob.perform_later(box)
+      end
+
+      # TODO ponastavovat v BetterUptime
+      BetterUptimeApi.ping_heartbeat('FS_SYNC')
```

Comment:
@celuchmarek pridame heartbeat? Mozem skusit aj ja ponastavovat, ale povedz, aby som ti nebabrala do toho pripadne.

Reply:
Jj, môžeš pridať a nastaviť. Todo comment vyhoď, to nastavíš hneď.